src/components/buttons/Button/Button.jsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Icon } from "../../icons/Icon/Icon";
+
+import "./button.scss";
+
+/**
+ * Button
+ *
+ * All buttons used in the app
+ *
+ * @return {jsx}
+ */
+export const Button = ({
+  /* Add props here */
```

Comment:
Is there any reason for this comment here?

Reply:
@georgiganchev-7DIGIT Please remove that.